skimage/measure/_regionprops.py

```diff
@@ -601,7 +603,25 @@ def perimeter(image, neighbourhood=4):
     .. [1] K. Benkrid, D. Crookes. Design and FPGA Implementation of
            a Perimeter Estimator. The Queen's University of Belfast.
            http://www.cs.qub.ac.uk/~d.crookes/webpubs/papers/perimeter.doc
+
+    Examples
+    --------
+    >>> from skimage import data, util
+    >>> from skimage.measure import label
+
+    # coins image (binary)
```

Comment:
I'm not sure this is correct syntax. Have you tried to check by building the documentation locally?

Reply:
I removed the blank line and add some `>>>`.

I get two warnings from the build : 
- WARNING: the sphinx_copybutton extension does not declare if it is safe for parallel reading, assuming it isn't - please ask the extension author to check and make it explicit
- WARNING: doing serial read

Except that, it seems ok.